tests/ui/parser/bad-char-literals.stderr

```diff
@@ -10,7 +10,7 @@ error: character constant must be escaped: `\n`
 LL |       '
    |  ______^
 LL | | ';
-   | |_ help: escape the character: `\n`
+   | |_^help: escape the character: `\n`
```

Comment:
Yeah. This is a pre-existing issue that I'm gonna have to figure out. The current change is "incidental", note that the previous output was also wrong. This is because there's a "zero-width but 1-line-tall char" involved, and the label layout algo doesn't account for that. I'd want to handle that in a separate PR (and consider the new output to be a slight improvement).

Reply:
Fixed